packages/scene-composer/src/components/panels/SceneHierarchyPanel/components/SceneHierarchyTree/SceneHierarchyTreeItem.tsx

```diff
@@ -81,7 +78,7 @@ const SceneHierarchyTreeItem: FC<SceneHierarchyTreeItemProps> = ({
       labelText={<SceneNodeLabel objectRef={key} labelText={labelText} componentTypes={componentTypes} />}
       onExpand={onExpandNode}
       expanded={expanded}
-      expandable={node && node.childRefs.length > 0 && !isSearching}
+      expandable={((node && node.childRefs.length > 0) || isValidModelRef) && !isSearching}
```

Comment:
is this change to show expand button for submodel tree? if so, we should use `showSubModel` to be consistent with when the tree is actually rendered

Reply:
This is taking into consideration the logic for `EnvironmentModel' as those aren't supposed to have expand/collapse functionality.

Reply:
so the problem your are trying to solve is when submodel tree is the only child item, the expand button is not shown. you should just need to handle this issue by showing the button when submodel tree is shown which is when `showSubModel` is true. 

`showSubModel` handles environment model and also viewing mode. with your change, the expand button will be shown when the model has no child at all in viewing mode

Reply:
so the problem your are trying to solve is when submodel tree is the only child item, the expand button is not shown. you should just need to handle this issue by showing the button when submodel tree is shown which is when `showSubModel` is true.

`showSubModel` handles environment model and also viewing mode. with your change, the expand button will be shown when the model has no child at all in viewing mode

Reply:
Incorrect. The problem that I am trying to solve is for the expand/collapse button to be shown when **any** child is present. Using `showSubModel` incorrectly associates this with only submodel logic, while `isValidModelRef` is more explicit as to what is/is not valid (submodel or not). 

Both changes work in composer & viewer which were tested on my end. Are you seeing different behavior on your's when pulling this branch locally? If so, please sync up with me so that we can determine why we would see inconsistent behavior. 

For the sake of readability & best practices, we should not have this explicitly tied to submodels.